readme.md

```diff
@@ -1,9 +1,13 @@
+# Sashimi has been moved
+
+As per the [Dependency Management work](https://docs.google.com/document/d/187L7C3oW7LKmPJoWTWvdtP7Ou-BjMPfF1BUh4Wcw-Po/edit#heading=h.tf0suvubpekj) this Sashimi now lives in the [Octopus Deploy Solution](https://github.com/OctopusDeploy/OctopusDeploy).
```

Comment:
Given this is a public repository, perhaps we could reword this to be a little more friendly to public-facing consumers. We can still link off to our private stuff, but just make a clear distinction between public and private.

Maybe something like:

> In 2022, we found that the costs of having Sashimi separated out across multiple repositories were outweighing the intended benefits, so we invested in reducing these development costs by consolidating our this Sashimi repository:
> * The "Calamari half" has been migrated to the [🌏consolidated Calamari Repository](https://github.com/OctopusDeploy/Calamari) (public)
> * The "Sashimi half" has been merged into the [🔒Octopus Server solution](https://github.com/OctopusDeploy) (private)
>
> For Octopus Deploy employees, further info on this work can be found in our [🔒 
Dependency Management documentation](https://docs.google.com/document/d/187L7C3oW7LKmPJoWTWvdtP7Ou-BjMPfF1BUh4Wcw-Po/edit#heading=h.tf0suvubpekj) (internal)